ovirt_imageio/client/_ovirt.py

```diff
@@ -47,6 +47,7 @@ def connect(args):
         username=args.username,
         password=args.password,
         ca_file=args.cafile,
+        insecure=not args.secure,
```

Comment:
This should fix communication with engine, and communication with the host should already work.

Reply:
Yes, I've tested the changed manually using the command `ovirt-img download-disk --engine-url=http://vm-10-122.lab.eng.tlv2.redhat.com --username=admin@internal --password-file=ovirt.pass --insecure  69f5d43f-e6c5-471c-8fe7-afac60e4e845 test` tried to initiate an Insecure transfer and it failed due to certificate issue, after I've added the above line the insecure transfer worked and the disk was downloaded successfully without any errors.

Reply:
and even better than this - it was tested with forklift! :)